src/dfu.cpp

```diff
@@ -45,21 +47,39 @@ void dfu_loop() {
         case DFU_STATE_NUKE:
             dfu_nuke();
             break;
+        case DFU_STATE_ACTIVATE:
+            dfu_activate();
+            break;
+        case DFU_STATE_RENEW_ADDRESS:
+            dfu_renew_address();
+            break;
+        case DFU_STATE_SEND_PACKET:
+            dfu_send_packet();
+            break;
     }
 }
 
+
+
+
+
 void dfu_help() {
     Serial.println(F("DFU Mode Instructions:"));
     Serial.println(F("S=setid       Set Node ID in EEPROM"));
     Serial.println(F("T=test        Test radio and print result"));
     Serial.println(F("N=nuke        Nuke the DFU for good"));
     Serial.println(F("H=human       Enter / Exit human mode; detailed response"));
+    Serial.println(F("A=activate    Activate the node"));
+    Serial.println(F("R=Renew       Renew node address"));
+    Serial.println(F("P=send        Send packet"));
     dfu_state = DFU_STATE_MAIN;
 }
 
 void dfu_main() {
     Serial.print(F("DFU>"));
-    while(!Serial.available());
+    while(!Serial.available()){
+        dfu_activate();
```

Comment:
Likewise (mentioned in a previous discussion), this should be things like `node.update()` rather than calling activate continuously.